internal/system/agent/init.go

```diff
@@ -65,7 +65,6 @@ func Retry(useConsul bool, useProfile string, timeout int, wait *sync.WaitGroup,
 
 		// Exit the loop if the dependencies have been satisfied.
 		if Configuration != nil {
-			executorClient, err = newExecutorClient()
```

Comment:
you should adjust the comment at line 36 to reflect that it's no longer set on initialization